nnpdf_data/nnpdf_data/new_commondata/PHENIX_1JET_200GEV_ALL/metadata.yaml

```
@@ -0,0 +1,52 @@
+setname: "PHENIX_1JET_200GEV_ALL"
```

```suggestion
setname: "PHENIX_1JET_200GEV"
```
I think this should be the consistent name, ie observable only in the FKtable name.

---

Yes, exactly! The observable name cannot be part of the folder name.